migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java

```diff
@@ -131,67 +151,72 @@ public void run() {
                     // Lets recalculate period, just in case it changed
                     period=parsePeriod(migrationConfiguration.getPeriod());
 
-                    // We abuse the migration job locking mechanism
-                    // here to make sure only one consistency checker
-                    // controller is up at any given time This process
-                    // only creates migration jobs for the new
-                    // periods, so this is not a high-load process,
-                    // and we don't need many instances of it
-                    // running. But since the migrator can run on
-                    // multiple hosts, there is no way to prevent
-                    // it. At least, we make sure only one of the
-                    // consistencyy checker controllers wakes up at
-                    // any given time.
-
-                    // We create a lock using a custom id:
-                    String lockId=migrationConfiguration.getConfigurationName()+":ConsistencyCheckerController";
-                    ActiveExecution ae;
-                    try {
-                        ae=lock(lockId);
-                    } catch (Exception e) {
-                        LOGGER.error("Exception during lock attempt {}:{}",lockId,e);
-                        ae=null;
-                    }
-                    if(ae!=null) {
-                        Breakpoint.checkpoint("CCC:locked");
+                    if(migrationJobsExist()) {
+                        LOGGER.error("There are migration jobs for {}, not running consistency checker this time",
```

Comment:
It is info now.

On Tue, Aug 25, 2015 at 8:45 AM, Marek <notifications@github.com> wrote:

> In
> migrator/src/main/java/com/redhat/lightblue/migrator/ConsistencyCheckerController.java
> <https://github.com/lightblue-platform/lightblue-migrator/pull/244#discussion_r37873653>
> :
>
> > -                    // consistencyy checker controllers wakes up at
> > -                    // any given time.
> > -
> > -                    // We create a lock using a custom id:
> > -                    String lockId=migrationConfiguration.getConfigurationName()+":ConsistencyCheckerController";
> > -                    ActiveExecution ae;
> > -                    try {
> > -                        ae=lock(lockId);
> > -                    } catch (Exception e) {
> > -                        LOGGER.error("Exception during lock attempt {}:{}",lockId,e);
> > -                        ae=null;
> > -                    }
> > -                    if(ae!=null) {
> > -                        Breakpoint.checkpoint("CCC:locked");
> > +                    if(migrationJobsExist()) {
> > +                        LOGGER.error("There are migration jobs for {}, not running consistency checker this time",
>
> This is normal operation, right? Info should be fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-migrator/pull/244/files#r37873653>
> .
>